HTML/mockup-mogo/js/task7.js

```diff
@@ -0,0 +1,66 @@
+import {promiseElements} from './elements.js';
+
+export const initSlider = () => {
+  promiseElements.then((data) => {
+    const {
+      opinionsPartFirst,
+      opinionsPartSecond,
+      buttonLeft,
+      buttonRight,
+    } = data;
+    let curIndexFirst = 0;
+    let curIndexSecond = 0;
+
+    buttonLeft.forEach((el, ind) => {
+      el.addEventListener('click', () => {
```

Comment:
it would be better to create generic function/class by passing arguments(for example container element) to which you will create new slider, as here you have too many hardcoded values and with some design changes you may break here everything